src/client/config/constants.ts

```diff
@@ -190,3 +190,5 @@ export const exportOptions: Array<{ label: string, fileFormat: FileFormat }> = [
   { label: STRINGS.exportToCSV, fileFormat: "csv" },
   { label: STRINGS.exportToTSV, fileFormat: "tsv" }
 ];
+
+export const copyToClipboardOptions = { debug: false, message: "Copy to clipboard: #{key}, Enter", format: "text/plain" };
```

Comment:
I see in the docs that debug and message are equal to defaults, so we should be able to not define them here?

Reply:
As per "node_modules/@types/react-copy-to-clipboard/index.d.ts", `debug` and `message` properties have to be defined if `options` object is passed.

```
declare namespace CopyToClipboard {
  interface Options {
    debug: boolean;
    message: string;
  }

  interface Props {
    text: string;
    onCopy?(a: string, b: boolean): void;
    options?: Options;
  }
}

declare class CopyToClipboard extends React.Component<CopyToClipboard.Props> {
}
```

Documentation from my initial comment is for latest version of the package so it's not entirely accurate, my bad.